### PR TITLE
Data transparency APIs added

### PR DESCRIPTION
**Pull request by RajaKsv**
Data transparency API specifications added

### CODE REVIEW

File: cat/CaT-service.yaml [hunk 1]
```diff
@@ -4153,12 +4150,102 @@ paths:
           $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Responses.yaml#/components/responses/500InternalServerError'
         '502':
           $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Responses.yaml#/components/responses/502BadGateway'
-      security:
-        - OAuth2:
-            - buyer
       summary: Get contract details
 
+
+  /public/projects:
```
Comment (ronin1972) [inline cat/CaT-service.yaml:4156]: Needs to be sat under Tenders. Can I suggest /tenders/public/projects. Better still it should be /tenders/projects and give the appropriate response based on having or not having a token

Comment (RajaKsv) [reply cat/CaT-service.yaml]: Addressed as part of the PR - https://github.com/Crown-Commercial-Service/ccs-scale-api-definitions/pull/224

File: cat/CaT-service.yaml [hunk 2]
```diff
@@ -4153,12 +4150,102 @@ paths:
...
+        '429':
+          $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Responses.yaml#/components/responses/429TooManyRequests'
+        '500':
+          $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Responses.yaml#/components/responses/500InternalServerError'
+        '502':
+          $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Responses.yaml#/components/responses/502BadGateway'
+      summary: Get project details for public
+
+
+  /public/projects/{project-id}:
```
[the remark this answers is not in the document]
Comment (RajaKsv) [reply cat/CaT-service.yaml]: As discussed, the existing get project API will be used. Additional request parameters should be added to limit the response information

File: cat/CaT-service.yaml [hunk 3]
```diff
@@ -4416,6 +4503,36 @@ components:
...
+        readOnly: true        
+
+    LotQueryParam:
+      name: lot-id
+      in: query
+      required: false
+      description: Lot
+      schema:
+        $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Schema.yaml#/components/schemas/LotId'
+
```
[the remark this answers is not in the document]
Comment (RajaKsv) [reply cat/CaT-service.yaml:4481]: Addressed as part of the PR - https://github.com/Crown-Commercial-Service/ccs-scale-api-definitions/pull/224

**Review by ronin1972 — APPROVED** [review]

**Pull request merged**